src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -89,6 +90,22 @@ private set
         private string content;
         private string name;
 
+
+        /// <summary>
+        /// Package Name
+        /// </summary>
+        internal string Name
```

Comment:
Maybe a more intuitive name would be PackageName... What do you think?

Reply:
Or graph name?

Reply:
@filipeotero yes, I think is a better idea to use PackageName, then updated in the next commit: https://github.com/DynamoDS/Dynamo/pull/13703/commits/0e3c928e47fc6f87b9d35cbe9b58b41d47318eda